examples/xml-access/xml-access.md

```diff
@@ -0,0 +1,17 @@
+# XML access
+
+It is possible to access elements in XML.
+
+- `x[i]` gives the `i`-th item (empty sequence if none).
+
+- `x.id` accesses the required attribute named `id`: the result is an error if there is no such attribute or if `x` is not a singleton.
+
+- `x?.id` accesses an optional attribute named `id`: the result is `()` if there is no such attribute. The `lang.xml` langlib provides the other operations.
+
+::: code xml_access.bal :::
+
+::: out xml_access.out :::
+
+## Related links
+- [XML data model](/learn/by-example/xml-data-model/)
```

Comment:
Let's add a link to `lang.xml` to related links

Reply:
Updated all BBEs that used `lang.xml`  in 98e746fb8d135223d968d98b71f7e3951834dc24